airflow/providers/amazon/PROVIDERS_CHANGES_2020.05.10.md

```diff
@@ -0,0 +1,78 @@
+| [3ea3e1a2b](https://github.com/apache/airflow/commit/3ea3e1a2b580b7ed10efe668de0cc37b03673500) | 2020-02-26  | [AIRFLOW-6824] EMRAddStepsOperator problem with multi-step XCom (#7443)                                                                                            |
+| [6eaa7e3b1](https://github.com/apache/airflow/commit/6eaa7e3b1845644d5ec65a00a997f4029bec9628) | 2020-02-25  | [AIRFLOW-5924] Automatically unify bucket name and key in S3Hook (#6574)                                                                                           |
+| [3320e432a](https://github.com/apache/airflow/commit/3320e432a129476dbc1c55be3b3faa3326a635bc) | 2020-02-24  | [AIRFLOW-6817] Lazy-load `airflow.DAG` to keep user-facing API untouched (#7517)                                                                                   |
+| [7d0e7122d](https://github.com/apache/airflow/commit/7d0e7122dd14576d834c6f66fe919a72b100b7f8) | 2020-02-24  | [AIRFLOW-6830] Add Subject/MessageAttributes to SNS hook and operator (#7451)                                                                                      |
+| [4d03e33c1](https://github.com/apache/airflow/commit/4d03e33c115018e30fa413c42b16212481ad25cc) | 2020-02-22  | [AIRFLOW-6817] remove imports from `airflow/__init__.py`, replaced implicit imports with explicit imports, added entry to `UPDATING.MD` - squashed/rebased (#7456) |
+| [47a922b86](https://github.com/apache/airflow/commit/47a922b86426968bfa07cc7892d2eeeca761d884) | 2020-02-21  | [AIRFLOW-6854] Fix missing typing_extensions on python 3.8 (#7474)                                                                                                 |
+| [9cbd7de6d](https://github.com/apache/airflow/commit/9cbd7de6d115795aba8bfb8addb060bfdfbdf87b) | 2020-02-18  | [AIRFLOW-6792] Remove _operator/_hook/_sensor in providers package and add tests (#7412)                                                                           |
+| [58c3542ed](https://github.com/apache/airflow/commit/58c3542ed25061320ce61dbe0adf451a44c738dd) | 2020-02-12  | [AIRFLOW-5231] Fix S3Hook.delete_objects method (#7375)                                                                                                            |
+| [b7aa778b3](https://github.com/apache/airflow/commit/b7aa778b38df2f116a1c20031e72fea8b97315bf) | 2020-02-10  | [AIRFLOW-6767] Correct name for default Athena workgroup (#7394)                                                                                                   |
+| [9282185e6](https://github.com/apache/airflow/commit/9282185e6624e64bb7f17447f81c1b2d1bb4d56d) | 2020-02-09  | [AIRFLOW-6761] Fix WorkGroup param in AWSAthenaHook (#7386)                                                                                                        |
```

Comment:
So. This commit is included in 1.10.10 already, which probably means we shouldn't include it here too....

:thinking:

Reply:
I thought about it, but I decided to include all the history from the "providers" package.

We can - (it's easy) to filter the cherry-picked ones out.

But I think it will be confusing - especially that someone might install it on an older version of Airflow (1.10.2 for example) where this change could yet not be cherry-picked. Then I see a lot of confusion (is this fix included or not)?